src/kinds.js

```diff
@@ -42,6 +42,17 @@ class StringKind extends Type {
 
 exports.StringKind = StringKind
 
+class IntKind extends Type {
```

Comment:
I wanted to test returning an integer so figured this was appropriate

Reply:
Yup, I had been meaning to do this.